cypress/e2e/Courses/Certificates/StandardPlanUnawardedCourseCertificate.feature

```diff
@@ -0,0 +1,26 @@
+Feature: Unawarded Premium Course Certificates for Standard Plan Users
+  Background:
+    Given a standard plan user has some completed unawarded premium courses
+    And the premium courses page has been navigated to
+
+  Scenario: Standard plan users should see the call to action "Review course" for completed and unawarded premium courses
+    Given an iOS mobile device is not being used
+    When the user selects the "Completed" tab
+    Then the user should see for unawarded courses cards the warning style
+    And should see the call to action "Review course"
+
+  Scenario: Standard plan users can successfully go to "Review course"
+    Given an iOS mobile device is not used
+    And the "Completed" tab is selected
+    And the user should see for unawarded courses cards the warning style
+    And the call to action "Review course" is displayed to the user
+    When the user click to "Review course"
+    Then the user should be taken to "Review course"
+
+  Scenario: Standard plan users with unawarded course credits can not download certificate
+    Given a standard plan users in the review course page
+    And the call to action "Download certificate" is not displayed to the user
+    And the call to action "Upgrade" is displayed to the user
+    And shows a copy text alert explaining the situation to the user
+    When the user requests to "Upgrade"
+    Then the user should be taken to plans page
```

Comment:
As I mentioned before we should assert that the contact support alert is shown, we should 

1. assert the exact text
2. assert that the link is correct 

For 2, something like this:
```js
  cy.get('a')
    .contains('contact support')
    .should('have.attr', 'href', 'https://acapedia.zendesk.com/hc/en-us/requests/new');
```

Reply:
As a general comment the original gherkin was quite big, it would have been splitted in two scenarios (we don't need it as standard cannot upgrade)

My wording is not correct, but I wanted to provide you a quick example of the two scenarios:

```
  Scenario: Standard plan users with unawarded course credits can not download certificate
    When a standard plan users access the review course page of an unawarded course
    Then the call to action "Download certificate" is not displayed to the user
    And the call to action "Upgrade" is displayed to the user
    And shows a copy text alert explaining the situation to the user

  Scenario: Standard plan users with unawarded course credits can upgrade to download the certificate
    Given a standard plan users is on the review course page of an unawarded course
    When the user requests to upgrade
    Then is redirected to my plan page
```